### PR TITLE
display not found on Rubric and Exemplar when rubric is empty, exempla…

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
…rs is empty and no answer key is found

avoids issue of displaying no information found and the answer key when an answer key is found along with empty rubric and exemplars

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.spec.ts [hunk 1]
```diff
@@ -38,14 +45,27 @@ describe('ItemExemplarComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
-});
 
+  it('should report not found when item scoring guide is empty', () => {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.spec.ts]: nit: test names don't read well

File: webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.ts [hunk 2]
```diff
@@ -37,7 +37,8 @@ export class ItemExemplarComponent implements OnInit {
 
             // TODO re-look at this logic
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.ts]: i knew this logic looked goofy months ago when i made this comment : p

Comment (esotrope) [reply webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.ts]: Does this change save us any network calls btw?

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.ts]: I don't think so. Before the issue was that we would display the answer key (on line 35) and the not found message (here). I don't think this impacts traffic though

File: webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.spec.ts (hunk 1 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.spec.ts]: suggestions? Jasmine naming is a little unfamiliar to me.

Comment (esotrope) [reply webapp/src/main/webapp/src/app/assessments/items/item-exemplar/item-exemplar.component.spec.ts]: Ah i see, "report" is a verb here. I think the real problem is i don't read well : j
I think i'd just replace the word "report" with "display" in these test names 👍

**Pull request merged**